docs/samples/shards/UI/Table/3.edn

```diff
@@ -0,0 +1,39 @@
+(defloop main-wire
+  (GFX.MainWindow
+   :Contents
+   (->
+    (Setup
+     (GFX.DrawQueue) >= .ui-draw-queue
+     (GFX.UIPass .ui-draw-queue) >> .render-steps)
+    .ui-draw-queue (GFX.ClearQueue)
+
+    (UI
+     .ui-draw-queue
+     (UI.CentralPanel
+      :Contents
+      (->
+       [{:Name "Doe" :Surname "John"}
+        {:Name "Dough" :Surname "Jane"}
+        {:Name "Smith" :Surname "Dick"}]
+       (UI.Table
+        :Resizable true
+        :Striped true
+        :RowIndex .index
+        :Columns
+        [{:Initial (float 20)}
+         {:Header "Surname"
+          :Initial (float 100) :AtLeast (float 60) :AtMost (float 160)}
+         {:Header
+          (-> "Name" (UI.Label :Style {:text_style "Heading"})
+              (UI.Button "Up" (Msg "Clicked Up") :Style {:text_style "Small"})
+              (UI.Button "Down" (Msg "Clicked Down") :Style {:text_style "Small"}))
+          :Initial (float 120) :AtLeast (float 100) :AtMost (float 160)}]
```

Comment:
a bit complex, are defaults decent?

Reply:
By default it is set to `egui_extras::Size::Remainder` which basically splits equally the columns. This can be seen in practice in the second sample `docs/samples/shards/UI/Table/2.edn`.